resources/lib/qobuz/registry.py

```diff
@@ -51,7 +51,6 @@ class Registry(object):
     def __init__(self, application):
         self.application = application
         try:
```

Comment:
import is needed to trigger the right backend loading

Reply:
oki
but it triggers a unused code in the linters

Reply:
That's the purpose of 
```python
 import xbmc  # pylint: disable=W0612
```
We don't use the module, just trying to load it. You can also write it
```python
 import xbmc as _
```
Because variable prefixed with _____ are unused variable

Reply:
great thx